cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -17,13 +17,29 @@
 package co.cask.cdap.spi.hbase;
 
 
+import java.io.Closeable;
 import java.io.IOException;
 import javax.annotation.Nullable;
 
 /**
  * Interface providing the HBase DDL operations.
  */
-public interface HBaseDDLExecutor {
+public interface HBaseDDLExecutor extends Closeable {
+
+  /**
+   * Initialize the {@link HBaseDDLExecutor} with the configuration object provided
+   * @param conf the configurations with which to initialize
+   * @param <T>
+   */
+  <T> void initialize(T conf);
```

Comment:
I think it's better to pass in some new Context object, as we can add methods to that Context without breaking this interface. That Context would have the getConfiguration() method. We can do this in a separate PR as well, when we add the ability to access configuration properties in the cconf.